clients/cli/src/main.rs

```diff
@@ -793,28 +776,79 @@ async fn command_display(config: &Config, command_config: DisplayCli) -> Command
             )
             .await?;
 
-        let mut displays = vec![];
-        for pool in pools {
+        let mut pool_and_vote_addresses = vec![];
+        for pool in pools.into_iter() {
             let vote_account_address =
                 try_from_slice_unchecked::<SinglePool>(&pool.1.data)?.vote_account_address;
-            displays.push(get_pool_display(config, pool.0, Some(vote_account_address)).await?);
+            pool_and_vote_addresses.push((pool.0, vote_account_address));
         }
 
-        Ok(format_output(
-            config,
-            "DisplayAll".to_string(),
-            StakePoolListOutput(displays),
-        ))
+        pool_and_vote_addresses
     } else {
         let pool_address = pool_address_from_args(
             command_config.pool_address,
             command_config.vote_account_address,
         );
 
+        vec![(
+            pool_address,
+            get_vote_address_from_pool(config, pool_address).await?,
+        )]
+    };
+
+    if pool_and_vote_addresses.len() > 100 {
+        return Err(
+            "Displaying more than 100 pools is not implemented; if you see \
+            this error, feel free to open an issue in the SVSP repo."
+                .into(),
+        );
+    }
```

Comment:
`getMultipleAccounts` limit. mainnet has 40-something pools, so it wont be relevant for awhile. ill probably do the >100 chuncking and the onramp support in display at the same time, the two of them will make it complicated enough ill refactor all this stuff into its own module